### ISSUE TITLE
[Feature Request]  Automatic loading of previous Memory Cards

### ISSUE THREAD

**Issue by noswanson1982** [aldostools/webMAN-MOD#1139]
I am enjoying the ability to load unencrypted ISO files without having to encrypt them for PS2 Classics Placeholder.  However, I have one grievance... they use the global Virtual Memory Cards found in the Memory Card Manager.

This would be fine, except for the annoying fact that whenever I switch from my user profile, to my wife's, we either have to share the same memory card (which results in the risk of data being overwritten), or having to manually switch slots on the memory cards.  As of right now, I have my PS2 memory card, my PSX memory card, my wife's PS2 memory card, and my wife's PSX memory card.  When my wife wants to play, she (not being as tech savvy as I am), will have to remember to go into the Memory Card Manager and manually switch her PS2 and PSX memory cards to Slot 1 and Slot 2 before she plays anything.  She would either find this inconvenient, or just not remember to do it at all.  This forces me to keep PS2 Classics Placeholder versions for her as those are saved within each user's own directory.

Would there be a way to detect when the profile gets switched and to reload that last used memory card configuration?  For instance, create a way to save to a cfg file the memory card assignments for each profile and then reload that file when the profile switches.  I think it would be a great to be able to load my profile and automatically have my PS2 and PSX memory cards already loaded, and then when my wife plays, she can switch to her profile, and the slots for her PS2 and PSX memory cards are automatically switched to her cards so she can just play.

Thanks for this great software.  I love what it is currently able to do!!

**Comment by aldostools** [comment]
I don't have a method to use the last memory card configuration.

However you could make a "shadow copy" of the virtual memory card from one profile in the other, so both files in different directories point to the same file. That way when you switch the profiles you will be using the same memory card.

The shadow copy can be done using IRISMAN. Press △ for the menu, then use LEFT/RIGHT buttons on the Copy option until you get Shadow Copy.

Also webMAN MOD web command /copy.ps3 uses symbolic links (shadow copy) when the files are copied from one folder in HDD0 to another.

**Comment by noswanson1982** [comment]
Yeah, I have two pairs of VM2 saves (A PS2 and a PSX for both myself and my wife).  I was hoping for a way to automate the switching between the two depending on which profile is loaded.  Either by reloading the last saved slots from a particular user's previous session, or to just be able to bind slots to a profile.  I was hoping to do anything BUT share slots between profiles.  I'm wondering how hard it would be to create something like that.

**Comment by aldostools** [comment]
webMAN MOD offers web commands for copy, swap, move, rename, delete files.
https://github.com/aldostools/webMAN-MOD/wiki/Web-Commands/#batch-scripts

You can use them to manually swap the files from XMB. You can use a bookmark in PS3 Internet Browser or create a Web Apps package to execute the web commands.

There is a onxmb.bat that is executed automatically when the user login or returns to XMB, but it's easier that you swap the files manually at demand.

**Comment by Joonie86** [comment]
No further feedback received
